db-init docker build failing

with
```
Run BRANCH="${GITHUB_REF_NAME//\//-}"
Unable to find image 'multiarch/qemu-user-static:latest' locally
latest: Pulling from multiarch/qemu-user-static
205dae5015e7: Pulling fs layer
816739e52091: Pulling fs layer
30abb83a18eb: Pulling fs layer
0657daef200b: Pulling fs layer
30c9c93f40b9: Pulling fs layer
0657daef200b: Waiting
30c9c93f40b9: Waiting
30abb83a18eb: Verifying Checksum
30abb83a18eb: Download complete
816739e52091: Verifying Checksum
816739e52091: Download complete
205dae5015e7: Verifying Checksum
205dae5015e7: Download complete
205dae5015e7: Pull complete
0657daef200b: Verifying Checksum
0657daef200b: Download complete
816739e52091: Pull complete
30abb83a18eb: Pull complete
0657daef200b: Pull complete
30c9c93f40b9: Verifying Checksum
30c9c93f40b9: Download complete
30c9c93f40b9: Pull complete
Digest: sha256:fe60359c92e86a43cc87b3d906006245f77bfc0565676b80004cc666e4feb9f0
Status: Downloaded newer image for multiarch/qemu-user-static:latest
Setting /usr/bin/qemu-alpha-static as binfmt interpreter for alpha
Setting /usr/bin/qemu-arm-static as binfmt interpreter for arm
Setting /usr/bin/qemu-armeb-static as binfmt interpreter for armeb
Setting /usr/bin/qemu-sparc-static as binfmt interpreter for sparc
Setting /usr/bin/qemu-sparc32plus-static as binfmt interpreter for sparc32plus
Setting /usr/bin/qemu-sparc64-static as binfmt interpreter for sparc64
Setting /usr/bin/qemu-ppc-static as binfmt interpreter for ppc
Setting /usr/bin/qemu-ppc64-static as binfmt interpreter for ppc64
Setting /usr/bin/qemu-ppc64le-static as binfmt interpreter for ppc64le
Setting /usr/bin/qemu-m68k-static as binfmt interpreter for m68k
Setting /usr/bin/qemu-mips-static as binfmt interpreter for mips
Setting /usr/bin/qemu-mipsel-static as binfmt interpreter for mipsel
Setting /usr/bin/qemu-mipsn32-static as binfmt interpreter for mipsn32
Setting /usr/bin/qemu-mipsn32el-static as binfmt interpreter for mipsn32el
Setting /usr/bin/qemu-mips64-static as binfmt interpreter for mips64
Setting /usr/bin/qemu-mips64el-static as binfmt interpreter for mips64el
Setting /usr/bin/qemu-sh4-static as binfmt interpreter for sh4
Setting /usr/bin/qemu-sh4eb-static as binfmt interpreter for sh4eb
Setting /usr/bin/qemu-s390x-static as binfmt interpreter for s390x
Setting /usr/bin/qemu-aarch64-static as binfmt interpreter for aarch64
Setting /usr/bin/qemu-aarch64_be-static as binfmt interpreter for aarch64_be
Setting /usr/bin/qemu-hppa-static as binfmt interpreter for hppa
Setting /usr/bin/qemu-riscv32-static as binfmt interpreter for riscv32
Setting /usr/bin/qemu-riscv64-static as binfmt interpreter for riscv64
Setting /usr/bin/qemu-xtensa-static as binfmt interpreter for xtensa
Setting /usr/bin/qemu-xtensaeb-static as binfmt interpreter for xtensaeb
Setting /usr/bin/qemu-microblaze-static as binfmt interpreter for microblaze
Setting /usr/bin/qemu-microblazeel-static as binfmt interpreter for microblazeel
Setting /usr/bin/qemu-or1k-static as binfmt interpreter for or1k
Setting /usr/bin/qemu-hexagon-static as binfmt interpreter for hexagon
#0 building with "default" instance using docker driver

#1 [internal] load build definition from arm64-Dockerfile
#1 transferring dockerfile: 746B done
#1 WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
#1 DONE 0.0s

#2 [auth] library/ubuntu:pull token for registry-1.docker.io
#2 DONE 0.0s

#3 [auth] treehouses/node-tags:pull token for registry-1.docker.io
#3 DONE 0.0s

#4 [internal] load metadata for docker.io/library/ubuntu:xenial
#4 ...

#5 [internal] load metadata for docker.io/treehouses/node-tags:arm64
#5 DONE 0.7s

#4 [internal] load metadata for docker.io/library/ubuntu:xenial
#4 DONE 0.8s

#6 [internal] load .dockerignore
#6 transferring context: 150B done
#6 DONE 0.0s

#7 [internal] load build context
#7 transferring context: 457.25kB done
#7 DONE 0.0s

#8 [stage-1 1/7] FROM docker.io/treehouses/node-tags:arm64@sha256:35c17611551ca9ab3efaf4e24b7f9ce542b476ea2b4a664db6654f599232d6a4
#8 resolve docker.io/treehouses/node-tags:arm64@sha256:35c17611551ca9ab3efaf4e24b7f9ce542b476ea2b4a664db6654f599232d6a4 done
#8 sha256:35c17611551ca9ab3efaf4e24b7f9ce542b476ea2b4a664db6654f599232d6a4 739B / 739B done
#8 sha256:8d62045e272bcfe1e4721870cdbfb801bd702d481a518f41195989e0faa604bd 819B / 819B done
#8 sha256:0eeab5c200691bd777e227c6eea27f7ca3c8232b67118a76edac2dcde3186aa1 0B / 2.72MB 0.1s
#8 sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f 0B / 17.01MB 0.1s
#8 sha256:0eeab5c200691bd777e227c6eea27f7ca3c8232b67118a76edac2dcde3186aa1 2.72MB / 2.72MB 0.2s done
#8 extracting sha256:0eeab5c200691bd777e227c6eea27f7ca3c8232b67118a76edac2dcde3186aa1
#8 extracting sha256:0eeab5c200691bd777e227c6eea27f7ca3c8232b67118a76edac2dcde3186aa1 0.1s done
#8 sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f 2.10MB / 17.01MB 0.8s
#8 sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f 6.29MB / 17.01MB 0.9s
#8 sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f 12.58MB / 17.01MB 1.1s
#8 sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f 17.01MB / 17.01MB 1.2s done
#8 extracting sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f
#8 extracting sha256:4c1eb0d98299a1b8e2c1a524065f15cef876af9b0260e433fcb2fe60d6cb360f 1.4s done
#8 DONE 3.0s

#9 [builder 1/3] FROM docker.io/library/ubuntu:xenial@sha256:1f1a2d56de1d604801a9671f301190704c25d604a416f59e03c04f5c6ffee0d6
#9 resolve docker.io/library/ubuntu:xenial@sha256:1f1a2d56de1d604801a9671f301190704c25d604a416f59e03c04f5c6ffee0d6 done
#9 sha256:1f1a2d56de1d604801a9671f301190704c25d604a416f59e03c04f5c6ffee0d6 1.42kB / 1.42kB done
#9 sha256:a3785f78ab8547ae2710c89e627783cfa7ee7824d3468cae6835c9f4eae23ff7 1.15kB / 1.15kB done
#9 sha256:b6f50765242581c887ff1acc2511fa2d885c52d8fb3ac8c4bba131fd86567f2e 3.36kB / 3.36kB done
#9 sha256:58690f9b18fca6469a14da4e212c96849469f9b1be6661d2342a4bf01774aa50 46.50MB / 46.50MB 0.9s done
#9 sha256:b51569e7c50720acf6860327847fe342a1afbe148d24c529fb81df105e3eed01 857B / 857B 0.3s done
#9 sha256:da8ef40b9ecabc2679fe2419957220c0272a965c5cf7e0269fa1aeeb8c56f2e1 528B / 528B 0.4s done
#9 sha256:fb15d46c38dcd1ea0b1990006c3366ecd10c79d374f341687eb2cb23a2c8672e 170B / 170B 0.5s done
#9 extracting sha256:58690f9b18fca6469a14da4e212c96849469f9b1be6661d2342a4bf01774aa50 1.7s done
#9 extracting sha256:b51569e7c50720acf6860327847fe342a1afbe148d24c529fb81df105e3eed01 done
#9 extracting sha256:da8ef40b9ecabc2679fe2419957220c0272a965c5cf7e0269fa1aeeb8c56f2e1 done
#9 extracting sha256:fb15d46c38dcd1ea0b1990006c3366ecd10c79d374f341687eb2cb23a2c8672e done
#9 DONE 3.0s

#10 [builder 2/3] COPY ./docker/db-init/crosscompile_db-init.sh .
#10 DONE 0.0s

#11 [builder 3/3] RUN bash ./crosscompile_db-init.sh armv8 install
#11 0.119 Building db-init for armv8
#11 ...

#12 [stage-1 2/7] RUN apk update ;    apk add --no-cache bash curl git jq ca-certificates;    mkdir -p /usr/local/lib/node_modules;     ln -s /usr/local/lib/node_modules/add-cors-to-couchdb/bin.js /usr/local/bin/add-cors-to-couchdb
#12 0.212 fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/aarch64/APKINDEX.tar.gz
#12 0.582 fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/aarch64/APKINDEX.tar.gz
#12 1.162 v3.12.12-45-g8116a127ec [http://dl-cdn.alpinelinux.org/alpine/v3.12/main]
#12 1.162 v3.12.12-46-g1df69604c4 [http://dl-cdn.alpinelinux.org/alpine/v3.12/community]
#12 1.162 OK: 12637 distinct packages available
#12 1.261 fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/aarch64/APKINDEX.tar.gz
#12 1.507 fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/aarch64/APKINDEX.tar.gz
#12 1.950 (1/11) Installing ncurses-terminfo-base (6.2_p20200523-r1)
#12 1.982 (2/11) Installing ncurses-libs (6.2_p20200523-r1)
#12 2.019 (3/11) Installing readline (8.0.4-r0)
#12 2.045 (4/11) Installing bash (5.0.17-r0)
#12 2.090 Executing bash-5.0.17-r0.post-install
#12 2.119 (5/11) Installing libcurl (7.79.1-r1)
#12 2.152 (6/11) Installing curl (7.79.1-r1)
#12 2.180 (7/11) Installing expat (2.2.10-r4)
#12 2.204 (8/11) Installing pcre2 (10.35-r0)
#12 2.237 (9/11) Installing git (2.26.3-r1)
#12 2.658 (10/11) Installing oniguruma (6.9.5-r2)
#12 2.691 (11/11) Installing jq (1.6-r1)
#12 2.724 Executing busybox-1.31.1-r22.trigger
#12 2.765 OK: 84 MiB in 33 packages
#12 DONE 2.9s

#11 [builder 3/3] RUN bash ./crosscompile_db-init.sh armv8 install
#11 4.814 Reading package lists...
#11 5.369 Building dependency tree...
#11 5.445 Reading state information...
#11 5.507 gnupg is already the newest version (1.4.20-1ubuntu3.3).
#11 5.507 The following additional packages will be installed:
#11 5.507   ca-certificates krb5-locales libasn1-8-heimdal libcurl3-gnutls libffi6
#11 5.507   libgmp10 libgnutls30 libgssapi-krb5-2 libgssapi3-heimdal libhcrypto4-heimdal
#11 5.507   libheimbase1-heimdal libheimntlm0-heimdal libhogweed4 libhx509-5-heimdal
#11 5.507   libidn11 libk5crypto3 libkeyutils1 libkrb5-26-heimdal libkrb5-3
#11 5.507   libkrb5support0 libldap-2.4-2 libnettle6 libp11-kit0 libroken18-heimdal
#11 5.507   librtmp1 libsasl2-2 libsasl2-modules libsasl2-modules-db libsqlite3-0
#11 5.508   libssl1.0.0 libtasn1-6 libwind0-heimdal openssl
#11 5.508 Suggested packages:
#11 5.508   gnutls-bin krb5-doc krb5-user libsasl2-modules-otp libsasl2-modules-ldap
#11 5.508   libsasl2-modules-sql libsasl2-modules-gssapi-mit
#11 5.508   | libsasl2-modules-gssapi-heimdal
#11 5.508 The following NEW packages will be installed:
#11 5.509   ca-certificates curl krb5-locales libasn1-8-heimdal libcurl3-gnutls libffi6
#11 5.509   libgmp10 libgnutls30 libgssapi-krb5-2 libgssapi3-heimdal libhcrypto4-heimdal
#11 5.509   libheimbase1-heimdal libheimntlm0-heimdal libhogweed4 libhx509-5-heimdal
#11 5.509   libidn11 libk5crypto3 libkeyutils1 libkrb5-26-heimdal libkrb5-3
#11 5.509   libkrb5support0 libldap-2.4-2 libnettle6 libp11-kit0 libroken18-heimdal
#11 5.509   librtmp1 libsasl2-2 libsasl2-modules libsasl2-modules-db libsqlite3-0
#11 5.509   libssl1.0.0 libtasn1-6 libwind0-heimdal openssl
#11 5.795 0 upgraded, 34 newly installed, 0 to remove and 2 not upgraded.
#11 5.795 Need to get 5332 kB of archives.
#11 5.795 After this operation, 19.0 MB of additional disk space will be used.
#11 5.795 Get:1 http://archive.ubuntu.com/ubuntu xenial/main amd64 libffi6 amd64 3.2.1-4 [17.8 kB]
#11 6.035 Get:2 http://archive.ubuntu.com/ubuntu xenial/main amd64 libgmp10 amd64 2:6.1.0+dfsg-2 [240 kB]
#11 6.441 Get:3 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libnettle6 amd64 3.2-1ubuntu0.16.04.2 [93.7 kB]
#11 6.492 Get:4 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libhogweed4 amd64 3.2-1ubuntu0.16.04.2 [136 kB]
#11 6.550 Get:5 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libidn11 amd64 1.32-3ubuntu1.2 [46.5 kB]
#11 6.567 Get:6 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libp11-kit0 amd64 0.23.2-5~ubuntu16.04.2 [107 kB]
#11 6.599 Get:7 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libtasn1-6 amd64 4.7-3ubuntu0.16.04.3 [43.5 kB]
#11 6.610 Get:8 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libgnutls30 amd64 3.4.10-4ubuntu1.9 [548 kB]
#11 6.710 Get:9 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsqlite3-0 amd64 3.11.0-1ubuntu1.5 [398 kB]
#11 6.757 Get:10 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libssl1.0.0 amd64 1.0.2g-1ubuntu4.20 [1083 kB]
#11 6.842 Get:11 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 openssl amd64 1.0.2g-1ubuntu4.20 [492 kB]
#11 6.870 Get:12 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 ca-certificates all 20210119~16.04.1 [148 kB]
#11 6.879 Get:13 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 krb5-locales all 1.13.2+dfsg-5ubuntu2.2 [13.7 kB]
#11 6.879 Get:14 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libroken18-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [41.4 kB]
#11 6.881 Get:15 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libasn1-8-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [174 kB]
#11 6.890 Get:16 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libkrb5support0 amd64 1.13.2+dfsg-5ubuntu2.2 [31.2 kB]
#11 6.891 Get:17 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libk5crypto3 amd64 1.13.2+dfsg-5ubuntu2.2 [81.2 kB]
#11 6.916 Get:18 http://archive.ubuntu.com/ubuntu xenial/main amd64 libkeyutils1 amd64 1.5.9-8ubuntu1 [9904 B]
#11 6.917 Get:19 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libkrb5-3 amd64 1.13.2+dfsg-5ubuntu2.2 [273 kB]
#11 7.034 Get:20 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libgssapi-krb5-2 amd64 1.13.2+dfsg-5ubuntu2.2 [120 kB]
#11 7.037 Get:21 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libhcrypto4-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [85.0 kB]
#11 7.039 Get:22 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libheimbase1-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [29.3 kB]
#11 7.039 Get:23 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libwind0-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [47.8 kB]
#11 7.040 Get:24 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libhx509-5-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [107 kB]
#11 7.042 Get:25 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libkrb5-26-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [202 kB]
#11 7.046 Get:26 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libheimntlm0-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [15.1 kB]
#11 7.047 Get:27 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libgssapi3-heimdal amd64 1.7~git20150920+dfsg-4ubuntu1.16.04.1 [96.1 kB]
#11 7.151 Get:28 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsasl2-modules-db amd64 2.1.26.dfsg1-14ubuntu0.2 [14.5 kB]
#11 7.152 Get:29 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsasl2-2 amd64 2.1.26.dfsg1-14ubuntu0.2 [48.7 kB]
#11 7.269 Get:30 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libldap-2.4-2 amd64 2.4.42+dfsg-2ubuntu3.13 [159 kB]
#11 7.272 Get:31 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 librtmp1 amd64 2.4+20151223.gitfa8646d-1ubuntu0.1 [54.4 kB]
#11 7.273 Get:32 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libcurl3-gnutls amd64 7.47.0-1ubuntu2.19 [189 kB]
#11 7.277 Get:33 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsasl2-modules amd64 2.1.26.dfsg1-14ubuntu0.2 [47.7 kB]
#11 7.278 Get:34 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 curl amd64 7.47.0-1ubuntu2.19 [139 kB]
#11 7.335 debconf: delaying package configuration, since apt-utils is not installed
#11 7.347 Fetched 5332 kB in 1s (3093 kB/s)
#11 7.360 Selecting previously unselected package libffi6:amd64.
#11 7.360 (Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 4785 files and directories currently installed.)
#11 7.365 Preparing to unpack .../libffi6_3.2.1-4_amd64.deb ...
#11 7.367 Unpacking libffi6:amd64 (3.2.1-4) ...
#11 7.382 Selecting previously unselected package libgmp10:amd64.
#11 7.384 Preparing to unpack .../libgmp10_2%3a6.1.0+dfsg-2_amd64.deb ...
#11 7.384 Unpacking libgmp10:amd64 (2:6.1.0+dfsg-2) ...
#11 7.413 Selecting previously unselected package libnettle6:amd64.
#11 7.414 Preparing to unpack .../libnettle6_3.2-1ubuntu0.16.04.2_amd64.deb ...
#11 7.415 Unpacking libnettle6:amd64 (3.2-1ubuntu0.16.04.2) ...
#11 7.435 Selecting previously unselected package libhogweed4:amd64.
#11 7.436 Preparing to unpack .../libhogweed4_3.2-1ubuntu0.16.04.2_amd64.deb ...
#11 7.436 Unpacking libhogweed4:amd64 (3.2-1ubuntu0.16.04.2) ...
#11 7.458 Selecting previously unselected package libidn11:amd64.
#11 7.459 Preparing to unpack .../libidn11_1.32-3ubuntu1.2_amd64.deb ...
#11 7.460 Unpacking libidn11:amd64 (1.32-3ubuntu1.2) ...
#11 7.476 Selecting previously unselected package libp11-kit0:amd64.
#11 7.477 Preparing to unpack .../libp11-kit0_0.23.2-5~ubuntu16.04.2_amd64.deb ...
#11 7.478 Unpacking libp11-kit0:amd64 (0.23.2-5~ubuntu16.04.2) ...
#11 7.500 Selecting previously unselected package libtasn1-6:amd64.
#11 7.501 Preparing to unpack .../libtasn1-6_4.7-3ubuntu0.16.04.3_amd64.deb ...
#11 7.502 Unpacking libtasn1-6:amd64 (4.7-3ubuntu0.16.04.3) ...
#11 7.519 Selecting previously unselected package libgnutls30:amd64.
#11 7.520 Preparing to unpack .../libgnutls30_3.4.10-4ubuntu1.9_amd64.deb ...
#11 7.521 Unpacking libgnutls30:amd64 (3.4.10-4ubuntu1.9) ...
#11 7.570 Selecting previously unselected package libsqlite3-0:amd64.
#11 7.571 Preparing to unpack .../libsqlite3-0_3.11.0-1ubuntu1.5_amd64.deb ...
#11 7.572 Unpacking libsqlite3-0:amd64 (3.11.0-1ubuntu1.5) ...
#11 7.614 Selecting previously unselected package libssl1.0.0:amd64.
#11 7.615 Preparing to unpack .../libssl1.0.0_1.0.2g-1ubuntu4.20_amd64.deb ...
#11 7.616 Unpacking libssl1.0.0:amd64 (1.0.2g-1ubuntu4.20) ...
#11 7.693 Selecting previously unselected package openssl.
#11 7.694 Preparing to unpack .../openssl_1.0.2g-1ubuntu4.20_amd64.deb ...
#11 7.695 Unpacking openssl (1.0.2g-1ubuntu4.20) ...
#11 7.751 Selecting previously unselected package ca-certificates.
#11 7.752 Preparing to unpack .../ca-certificates_20210119~16.04.1_all.deb ...
#11 7.753 Unpacking ca-certificates (20210119~16.04.1) ...
#11 7.792 Selecting previously unselected package krb5-locales.
#11 7.793 Preparing to unpack .../krb5-locales_1.13.2+dfsg-5ubuntu2.2_all.deb ...
#11 7.794 Unpacking krb5-locales (1.13.2+dfsg-5ubuntu2.2) ...
#11 7.808 Selecting previously unselected package libroken18-heimdal:amd64.
#11 7.809 Preparing to unpack .../libroken18-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 7.810 Unpacking libroken18-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 7.826 Selecting previously unselected package libasn1-8-heimdal:amd64.
#11 7.827 Preparing to unpack .../libasn1-8-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 7.828 Unpacking libasn1-8-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 7.854 Selecting previously unselected package libkrb5support0:amd64.
#11 7.855 Preparing to unpack .../libkrb5support0_1.13.2+dfsg-5ubuntu2.2_amd64.deb ...
#11 7.856 Unpacking libkrb5support0:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 7.872 Selecting previously unselected package libk5crypto3:amd64.
#11 7.873 Preparing to unpack .../libk5crypto3_1.13.2+dfsg-5ubuntu2.2_amd64.deb ...
#11 7.874 Unpacking libk5crypto3:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 7.892 Selecting previously unselected package libkeyutils1:amd64.
#11 7.893 Preparing to unpack .../libkeyutils1_1.5.9-8ubuntu1_amd64.deb ...
#11 7.894 Unpacking libkeyutils1:amd64 (1.5.9-8ubuntu1) ...
#11 7.908 Selecting previously unselected package libkrb5-3:amd64.
#11 7.909 Preparing to unpack .../libkrb5-3_1.13.2+dfsg-5ubuntu2.2_amd64.deb ...
#11 7.910 Unpacking libkrb5-3:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 7.946 Selecting previously unselected package libgssapi-krb5-2:amd64.
#11 7.947 Preparing to unpack .../libgssapi-krb5-2_1.13.2+dfsg-5ubuntu2.2_amd64.deb ...
#11 7.948 Unpacking libgssapi-krb5-2:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 7.970 Selecting previously unselected package libhcrypto4-heimdal:amd64.
#11 7.971 Preparing to unpack .../libhcrypto4-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 7.972 Unpacking libhcrypto4-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 7.991 Selecting previously unselected package libheimbase1-heimdal:amd64.
#11 7.992 Preparing to unpack .../libheimbase1-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 7.993 Unpacking libheimbase1-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.008 Selecting previously unselected package libwind0-heimdal:amd64.
#11 8.009 Preparing to unpack .../libwind0-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 8.010 Unpacking libwind0-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.028 Selecting previously unselected package libhx509-5-heimdal:amd64.
#11 8.029 Preparing to unpack .../libhx509-5-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 8.030 Unpacking libhx509-5-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.050 Selecting previously unselected package libkrb5-26-heimdal:amd64.
#11 8.051 Preparing to unpack .../libkrb5-26-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 8.052 Unpacking libkrb5-26-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.078 Selecting previously unselected package libheimntlm0-heimdal:amd64.
#11 8.079 Preparing to unpack .../libheimntlm0-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 8.080 Unpacking libheimntlm0-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.094 Selecting previously unselected package libgssapi3-heimdal:amd64.
#11 8.095 Preparing to unpack .../libgssapi3-heimdal_1.7~git20150920+dfsg-4ubuntu1.16.04.1_amd64.deb ...
#11 8.096 Unpacking libgssapi3-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.114 Selecting previously unselected package libsasl2-modules-db:amd64.
#11 8.115 Preparing to unpack .../libsasl2-modules-db_2.1.26.dfsg1-14ubuntu0.2_amd64.deb ...
#11 8.116 Unpacking libsasl2-modules-db:amd64 (2.1.26.dfsg1-14ubuntu0.2) ...
#11 8.130 Selecting previously unselected package libsasl2-2:amd64.
#11 8.130 Preparing to unpack .../libsasl2-2_2.1.26.dfsg1-14ubuntu0.2_amd64.deb ...
#11 8.131 Unpacking libsasl2-2:amd64 (2.1.26.dfsg1-14ubuntu0.2) ...
#11 8.148 Selecting previously unselected package libldap-2.4-2:amd64.
#11 8.149 Preparing to unpack .../libldap-2.4-2_2.4.42+dfsg-2ubuntu3.13_amd64.deb ...
#11 8.150 Unpacking libldap-2.4-2:amd64 (2.4.42+dfsg-2ubuntu3.13) ...
#11 8.174 Selecting previously unselected package librtmp1:amd64.
#11 8.175 Preparing to unpack .../librtmp1_2.4+20151223.gitfa8646d-1ubuntu0.1_amd64.deb ...
#11 8.176 Unpacking librtmp1:amd64 (2.4+20151223.gitfa8646d-1ubuntu0.1) ...
#11 8.192 Selecting previously unselected package libcurl3-gnutls:amd64.
#11 8.193 Preparing to unpack .../libcurl3-gnutls_7.47.0-1ubuntu2.19_amd64.deb ...
#11 8.194 Unpacking libcurl3-gnutls:amd64 (7.47.0-1ubuntu2.19) ...
#11 8.219 Selecting previously unselected package libsasl2-modules:amd64.
#11 8.220 Preparing to unpack .../libsasl2-modules_2.1.26.dfsg1-14ubuntu0.2_amd64.deb ...
#11 8.220 Unpacking libsasl2-modules:amd64 (2.1.26.dfsg1-14ubuntu0.2) ...
#11 8.237 Selecting previously unselected package curl.
#11 8.238 Preparing to unpack .../curl_7.47.0-1ubuntu2.19_amd64.deb ...
#11 8.239 Unpacking curl (7.47.0-1ubuntu2.19) ...
#11 8.255 Processing triggers for libc-bin (2.23-0ubuntu11.3) ...
#11 8.276 Setting up libffi6:amd64 (3.2.1-4) ...
#11 8.279 Setting up libgmp10:amd64 (2:6.1.0+dfsg-2) ...
#11 8.281 Setting up libnettle6:amd64 (3.2-1ubuntu0.16.04.2) ...
#11 8.283 Setting up libhogweed4:amd64 (3.2-1ubuntu0.16.04.2) ...
#11 8.286 Setting up libidn11:amd64 (1.32-3ubuntu1.2) ...
#11 8.288 Setting up libp11-kit0:amd64 (0.23.2-5~ubuntu16.04.2) ...
#11 8.290 Setting up libtasn1-6:amd64 (4.7-3ubuntu0.16.04.3) ...
#11 8.293 Setting up libgnutls30:amd64 (3.4.10-4ubuntu1.9) ...
#11 8.295 Setting up libsqlite3-0:amd64 (3.11.0-1ubuntu1.5) ...
#11 8.301 Setting up libssl1.0.0:amd64 (1.0.2g-1ubuntu4.20) ...
#11 8.352 debconf: unable to initialize frontend: Dialog
#11 8.352 debconf: (TERM is not set, so the dialog frontend is not usable.)
#11 8.352 debconf: falling back to frontend: Readline
#11 8.352 debconf: unable to initialize frontend: Readline
#11 8.352 debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.22.1 /usr/local/share/perl/5.22.1 /usr/lib/x86_64-linux-gnu/perl5/5.22 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.22 /usr/share/perl/5.22 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base .) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
#11 8.352 debconf: falling back to frontend: Teletype
#11 8.368 Setting up openssl (1.0.2g-1ubuntu4.20) ...
#11 8.372 Setting up ca-certificates (20210119~16.04.1) ...
#11 8.425 debconf: unable to initialize frontend: Dialog
#11 8.425 debconf: (TERM is not set, so the dialog frontend is not usable.)
#11 8.425 debconf: falling back to frontend: Readline
#11 8.425 debconf: unable to initialize frontend: Readline
#11 8.425 debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.22.1 /usr/local/share/perl/5.22.1 /usr/lib/x86_64-linux-gnu/perl5/5.22 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.22 /usr/share/perl/5.22 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base .) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
#11 8.425 debconf: falling back to frontend: Teletype
#11 8.771 Setting up krb5-locales (1.13.2+dfsg-5ubuntu2.2) ...
#11 8.773 Setting up libroken18-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.775 Setting up libasn1-8-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.777 Setting up libkrb5support0:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 8.780 Setting up libk5crypto3:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 8.783 Setting up libkeyutils1:amd64 (1.5.9-8ubuntu1) ...
#11 8.785 Setting up libkrb5-3:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 8.787 Setting up libgssapi-krb5-2:amd64 (1.13.2+dfsg-5ubuntu2.2) ...
#11 8.791 Setting up libhcrypto4-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.793 Setting up libheimbase1-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.796 Setting up libwind0-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.799 Setting up libhx509-5-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.801 Setting up libkrb5-26-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.803 Setting up libheimntlm0-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.805 Setting up libgssapi3-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
#11 8.808 Setting up libsasl2-modules-db:amd64 (2.1.26.dfsg1-14ubuntu0.2) ...
#11 8.810 Setting up libsasl2-2:amd64 (2.1.26.dfsg1-14ubuntu0.2) ...
#11 8.812 Setting up libldap-2.4-2:amd64 (2.4.42+dfsg-2ubuntu3.13) ...
#11 8.819 Setting up librtmp1:amd64 (2.4+20151223.gitfa8646d-1ubuntu0.1) ...
#11 8.821 Setting up libcurl3-gnutls:amd64 (7.47.0-1ubuntu2.19) ...
#11 8.823 Setting up libsasl2-modules:amd64 (2.1.26.dfsg1-14ubuntu0.2) ...
#11 8.827 Setting up curl (7.47.0-1ubuntu2.19) ...
#11 8.830 Processing triggers for libc-bin (2.23-0ubuntu11.3) ...
#11 8.836 Processing triggers for ca-certificates (20210119~16.04.1) ...
#11 8.840 Updating certificates in /etc/ssl/certs...
#11 10.03 129 added, 0 removed; done.
#11 10.03 Running hooks in /etc/ca-certificates/update.d...
#11 10.03 done.
#11 10.29 
#11 10.29 ================================================================================
#11 10.29 ================================================================================
#11 10.29 
#11 10.29                               DEPRECATION WARNING                            
#11 10.29 
#11 10.29   Node.js 10.x is no longer actively supported!
#11 10.29 
#11 10.29   You will not receive security or critical stability updates for this version.
#11 10.29 
#11 10.29   You should migrate to a supported version of Node.js as soon as possible.
#11 10.29   Use the installation script that corresponds to the version of Node.js you
#11 10.29   wish to install. e.g.
#11 10.29 
#11 10.29    * https://deb.nodesource.com/setup_16.x — Node.js 16 "Gallium"
#11 10.29    * https://deb.nodesource.com/setup_18.x — Node.js 18 LTS "Hydrogen" (recommended)
#11 10.29    * https://deb.nodesource.com/setup_19.x — Node.js 19 "Nineteen"
#11 10.29    * https://deb.nodesource.com/setup_20.x — Node.js 20 "Iron" (current)
#11 10.29 
#11 10.29   Please see https://github.com/nodejs/Release for details about which
#11 10.29   version may be appropriate for you.
#11 10.29 
#11 10.29   The NodeSource Node.js distributions repository contains
#11 10.29   information both about supported versions of Node.js and supported Linux
#11 10.29   distributions. To learn more about usage, see the repository:
#11 10.29     https://github.com/nodesource/distributions
#11 10.29 
#11 10.29 ================================================================================
#11 10.29 ================================================================================
#11 10.29 
#11 10.29 Continuing in 20 seconds ...
#11 10.29 
#11 30.29 
#11 30.29 ================================================================================
#11 30.29 ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
#11 30.29 ================================================================================
#11 30.29 
#11 30.29                            SCRIPT DEPRECATION WARNING                    
#11 30.29 
#11 30.29   
#11 30.29   This script, located at https://deb.nodesource.com/setup_X, used to
#11 30.29   install Node.js is deprecated now and will eventually be made inactive.
#11 30.29 
#11 30.29   Please visit the NodeSource distributions Github and follow the
#11 30.29   instructions to migrate your repo.
#11 30.29   https://github.com/nodesource/distributions
#11 30.29 
#11 30.29   The NodeSource Node.js Linux distributions GitHub repository contains
#11 30.29   information about which versions of Node.js and which Linux distributions
#11 30.29   are supported and how to install it.
#11 30.29   https://github.com/nodesource/distributions
#11 30.29 
#11 30.29 
#11 30.29                           SCRIPT DEPRECATION WARNING
#11 30.29 
#11 30.29 ================================================================================
#11 30.29 ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
#11 30.29 ================================================================================
#11 30.29 
#11 30.29 TO AVOID THIS WAIT MIGRATE THE SCRIPT
#11 30.29 Continuing in 60 seconds (press Ctrl-C to abort) ...
#11 30.29 
#11 90.29 
#11 90.29 ## Installing the NodeSource Node.js 10.x repo...
#11 90.29 
#11 90.29 
#11 90.29 ## Populating apt-get cache...
#11 90.29 
#11 90.29 + apt-get update
#11 90.55 Hit:1 http://archive.ubuntu.com/ubuntu xenial InRelease
#11 90.66 Hit:2 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
#11 90.72 Hit:3 http://security.ubuntu.com/ubuntu xenial-security InRelease
#11 90.78 Hit:4 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
#11 90.86 Reading package lists...
#11 91.42 
#11 91.42 ## Installing packages required for setup: apt-transport-https lsb-release...
#11 91.42 
#11 91.42 + apt-get install -y apt-transport-https lsb-release > /dev/null 2>&1
#11 95.24 
#11 95.24 ## Confirming "xenial" is supported...
#11 95.24 
#11 95.24 + curl -sLf -o /dev/null 'https://deb.nodesource.com/node_10.x/dists/xenial/Release'
#11 95.80 
#11 95.80 ## Adding the NodeSource signing key to your keyring...
#11 95.80 
#11 95.80 + curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | tee /usr/share/keyrings/nodesource.gpg >/dev/null
#11 96.02 
#11 96.02 ## Creating apt sources list file for the NodeSource Node.js 10.x repo...
#11 96.02 
#11 96.02 + echo 'deb [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_10.x xenial main' > /etc/apt/sources.list.d/nodesource.list
#11 96.02 + echo 'deb-src [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_10.x xenial main' >> /etc/apt/sources.list.d/nodesource.list
#11 96.03 
#11 96.03 ## Running `apt-get update` for you...
#11 96.03 
#11 96.03 + apt-get update
#11 96.16 Hit:1 http://archive.ubuntu.com/ubuntu xenial InRelease
#11 96.21 Hit:2 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
#11 96.26 Hit:3 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
#11 96.26 Get:4 https://deb.nodesource.com/node_10.x xenial InRelease [4584 B]
#11 96.35 Hit:5 http://security.ubuntu.com/ubuntu xenial-security InRelease
#11 96.40 Ign:4 https://deb.nodesource.com/node_10.x xenial InRelease
#11 96.62 Get:6 https://deb.nodesource.com/node_10.x xenial/main amd64 Packages [767 B]
#11 96.63 Fetched 5351 B in 0s (9391 B/s)
#11 96.63 Reading package lists...
#11 97.18 W: GPG error: https://deb.nodesource.com/node_10.x xenial InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1655A0AB68576280
#11 97.18 W: The repository 'https://deb.nodesource.com/node_10.x xenial InRelease' is not signed.
#11 97.18 
#11 97.18 ## Run `sudo apt-get install -y nodejs` to install Node.js 10.x and npm
#11 97.18 ## You may also need development tools to build native addons:
#11 97.18      sudo apt-get install gcc g++ make
#11 97.18 ## To install the Yarn package manager, run:
#11 97.18      curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | sudo tee /usr/share/keyrings/yarnkey.gpg >/dev/null
#11 97.18      echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
#11 97.18      sudo apt-get update && sudo apt-get install yarn
#11 97.18 
#11 97.18 
#11 97.19 Reading package lists...
#11 97.74 Building dependency tree...
#11 97.82 Reading state information...
#11 97.88 The following additional packages will be installed:
#11 97.88   binutils binutils-aarch64-linux-gnu bzip2 cpp cpp-4.8-aarch64-linux-gnu
#11 97.88   cpp-5 dpkg-dev fakeroot g++ g++-5 gcc gcc-4.8-aarch64-linux-gnu-base gcc-5
#11 97.88   gcc-5-cross-base ifupdown iproute2 isc-dhcp-client isc-dhcp-common
#11 97.88   libalgorithm-diff-perl libalgorithm-diff-xs-perl libalgorithm-merge-perl
#11 97.88   libasan2 libatm1 libatomic1 libatomic1-arm64-cross libc-dev-bin
#11 97.88   libc6-arm64-cross libc6-dev libc6-dev-arm64-cross libcc1-0 libcilkrts5
#11 97.88   libcloog-isl4 libdns-export162 libdpkg-perl libfakeroot
#11 97.88   libfile-fcntllock-perl libgcc-4.8-dev-arm64-cross libgcc-5-dev
#11 97.88   libgcc1-arm64-cross libgdbm3 libgomp1 libgomp1-arm64-cross libisc-export160
#11 97.88   libisl15 libitm1 libitm1-arm64-cross liblsan0 libmnl0 libmpc3 libmpfr4
#11 97.88   libmpx0 libperl5.22 libpython-stdlib libpython2.7-minimal
#11 97.88   libpython2.7-stdlib libquadmath0 libstdc++-4.8-dev-arm64-cross
#11 97.88   libstdc++-5-dev libstdc++6-arm64-cross libtsan0 libubsan0 libxtables11
#11 97.88   linux-libc-dev linux-libc-dev-arm64-cross make manpages manpages-dev netbase
#11 97.88   patch perl perl-modules-5.22 python python-minimal python2.7
#11 97.88   python2.7-minimal rename xz-utils
#11 97.88 Suggested packages:
#11 97.88   binutils-doc bzip2-doc cpp-doc gcc-4.8-locales gcc-5-locales debian-keyring
#11 97.88   g++-multilib gcc-4.8-doc libstdc++6-4.8-dbg-arm64-cross g++-5-multilib
#11 97.88   gcc-5-doc libstdc++6-5-dbg gcc-multilib autoconf automake libtool flex bison
#11 97.88   gdb gcc-doc libgcc1-dbg-arm64-cross libgomp1-dbg-arm64-cross
#11 97.88   libitm1-dbg-arm64-cross libatomic1-dbg-arm64-cross libasan0-dbg-arm64-cross
#11 97.88   libtsan0-dbg-arm64-cross libquadmath0-dbg-arm64-cross gcc-5-multilib
#11 97.88   libgcc1-dbg libgomp1-dbg libitm1-dbg libatomic1-dbg libasan2-dbg
#11 97.88   liblsan0-dbg libtsan0-dbg libubsan0-dbg libcilkrts5-dbg libmpx0-dbg
#11 97.88   libquadmath0-dbg ppp rdnssd iproute2-doc resolvconf avahi-autoipd
#11 97.88   isc-dhcp-client-ddns apparmor glibc-doc libstdc++-5-doc make-doc man-browser
#11 97.88   ed diffutils-doc perl-doc libterm-readline-gnu-perl
#11 97.88   | libterm-readline-perl-perl python-doc python-tk python2.7-doc
#11 97.88   binfmt-support
#11 97.88 The following NEW packages will be installed:
#11 97.88   binutils binutils-aarch64-linux-gnu build-essential bzip2 cpp
#11 97.88   cpp-4.8-aarch64-linux-gnu cpp-5 dpkg-dev fakeroot g++
#11 97.88   g++-4.8-aarch64-linux-gnu g++-5 gcc gcc-4.8-aarch64-linux-gnu
#11 97.88   gcc-4.8-aarch64-linux-gnu-base gcc-5 gcc-5-cross-base ifupdown iproute2
#11 97.88   isc-dhcp-client isc-dhcp-common libalgorithm-diff-perl
#11 97.88   libalgorithm-diff-xs-perl libalgorithm-merge-perl libasan2 libatm1
#11 97.88   libatomic1 libatomic1-arm64-cross libc-dev-bin libc6-arm64-cross libc6-dev
#11 97.88   libc6-dev-arm64-cross libcc1-0 libcilkrts5 libcloog-isl4 libdns-export162
#11 97.88   libdpkg-perl libfakeroot libfile-fcntllock-perl libgcc-4.8-dev-arm64-cross
#11 97.88   libgcc-5-dev libgcc1-arm64-cross libgdbm3 libgomp1 libgomp1-arm64-cross
#11 97.88   libisc-export160 libisl15 libitm1 libitm1-arm64-cross liblsan0 libmnl0
#11 97.88   libmpc3 libmpfr4 libmpx0 libperl5.22 libpython-stdlib libpython2.7-minimal
#11 97.88   libpython2.7-stdlib libquadmath0 libstdc++-4.8-dev-arm64-cross
#11 97.88   libstdc++-5-dev libstdc++6-arm64-cross libtsan0 libubsan0 libxtables11
#11 97.88   linux-libc-dev linux-libc-dev-arm64-cross make manpages manpages-dev netbase
#11 97.88   nodejs patch perl perl-modules-5.22 python python-minimal python2.7
#11 97.88   python2.7-minimal rename xz-utils
#11 98.00 0 upgraded, 81 newly installed, 0 to remove and 2 not upgraded.
#11 98.00 Need to get 88.7 MB of archives.
#11 98.00 After this operation, 362 MB of additional disk space will be used.
#11 98.00 WARNING: The following packages cannot be authenticated!
#11 98.00   nodejs
#11 98.00 E: There were unauthenticated packages and -y was used without --allow-unauthenticated
#11 98.00 ./crosscompile_db-init.sh: line 29: npm: command not found
#11 ERROR: process "/bin/sh -c bash ./crosscompile_db-init.sh armv8 install" did not complete successfully: exit code: 127
------
 > [builder 3/3] RUN bash ./crosscompile_db-init.sh armv8 install:
97.88   linux-libc-dev linux-libc-dev-arm64-cross make manpages manpages-dev netbase
97.88   nodejs patch perl perl-modules-5.22 python python-minimal python2.7
97.88   python2.7-minimal rename xz-utils
98.00 0 upgraded, 81 newly installed, 0 to remove and 2 not upgraded.
98.00 Need to get 88.7 MB of archives.
98.00 After this operation, 362 MB of additional disk space will be used.
98.00 WARNING: The following packages cannot be authenticated!
98.00   nodejs
98.00 E: There were unauthenticated packages and -y was used without --allow-unauthenticated
98.00 ./crosscompile_db-init.sh: line 29: npm: command not found
------

 3 warnings found (use docker --debug to expand):
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
 - InvalidBaseImagePlatform: Base image treehouses/node-tags:arm64 was pulled with platform "linux/arm64", expected "linux/amd64" for current build (line 9)
 - JSONArgsRecommended: JSON arguments recommended for CMD to prevent unintended behavior related to OS signals (line 24)
arm64-Dockerfile:5
--------------------
   3 |     
   4 |     COPY ./docker/db-init/crosscompile_db-init.sh .
   5 | >>> RUN bash ./crosscompile_db-init.sh armv8 install
   6 |     
   7 |     #####
--------------------
ERROR: failed to build: failed to solve: process "/bin/sh -c bash ./crosscompile_db-init.sh armv8 install" did not complete successfully: exit code: 127
Error: Process completed with exit code 1.
```

--- a/docker/db-init/crosscompile_db-init.sh
+++ b/docker/db-init/crosscompile_db-init.sh
@@ -25,7 +25,7 @@ if [[ "${ACT}" == "install" ]]; then
   apt-get update -qq
   apt-get install -y curl gnupg
   curl -sL https://deb.nodesource.com/setup_10.x | bash -
-  apt-get install -y nodejs build-essential ${PACKAGES}
+  apt-get install -y --allow-unauthenticated nodejs build-essential ${PACKAGES}
   npm install "--arch=${TRIPLE}" -g add-cors-to-couchdb
 else
   echo "Error: No action Specified"

--- a/docker/db-init/crosscompile_db-init.sh
+++ b/docker/db-init/crosscompile_db-init.sh
@@ -23,11 +23,10 @@ echo "Building db-init for ${ARCH}"
 
 if [[ "${ACT}" == "install" ]]; then
   apt-get update -qq
-  apt-get install -y curl gnupg apt-transport-https
-  # Import NodeSource GPG key explicitly
-  curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /usr/share/keyrings/nodesource.gpg || true
+  apt-get install -y curl gnupg
   curl -sL https://deb.nodesource.com/setup_10.x | bash -
-  apt-get update -qq
+  # Use --allow-unauthenticated since Node.js 10.x repository GPG key is deprecated/unavailable
+  # This is acceptable as Node.js 10 itself is EOL and this is only for cross-compilation build
   apt-get install -y --allow-unauthenticated nodejs build-essential ${PACKAGES}
   npm install "--arch=${TRIPLE}" -g add-cors-to-couchdb
 else

--- a/docker/db-init/crosscompile_db-init.sh
+++ b/docker/db-init/crosscompile_db-init.sh
@@ -23,8 +23,11 @@ echo "Building db-init for ${ARCH}"
 
 if [[ "${ACT}" == "install" ]]; then
   apt-get update -qq
-  apt-get install -y curl gnupg
+  apt-get install -y curl gnupg apt-transport-https
+  # Import NodeSource GPG key explicitly
+  curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /usr/share/keyrings/nodesource.gpg || true
   curl -sL https://deb.nodesource.com/setup_10.x | bash -
+  apt-get update -qq
   apt-get install -y --allow-unauthenticated nodejs build-essential ${PACKAGES}
   npm install "--arch=${TRIPLE}" -g add-cors-to-couchdb
 else

--- a/package.json
+++ b/package.json
@@ -1,9 +1,9 @@
 {
   "name": "planet",
   "license": "AGPL-3.0",
-  "version": "0.21.29",
+  "version": "0.21.30",
   "myplanet": {
-    "latest": "v0.43.89",
+    "latest": "v0.44.48",
     "min": "v0.37.60"
   },
   "scripts": {

